src/mpc-hc/mpcresources/PO/mpc-hc.fr.strings.po

```diff
@@ -4269,7 +4269,7 @@ msgstr "/dvdpos T#C\tDémarre la lecture du titre T, à partir du chapitre C"
 msgctxt "IDS_CMD_DVDPOS_TIME"
 msgid "/dvdpos T#P\tStart playback at title T, position P (hh:mm:ss)"
 msgstr ""
-"/dvdpos T#P\tDémarrer la lecture du titre T, à partir de la position P "
+"/dvdpos T#P⇥Démarrer la lecture du titre T, à partir de la position P "
```

Comment:
This one and one below are wrong.

Reply:
What is that, a unicode tab or something?

Reply:
It's U+21e5.  Weird.  It shows a tab in transifex, but that file isn't 100% so it didn't sync yet.  Forcing a sync.